src/components/EditImageModal/index.jsx

```diff
@@ -1,44 +1,61 @@
 import React from 'react';
-import { Button, CheckBox, Icon, InputText, Modal } from '@edx/paragon';
+import { Button, CheckBox, Icon, InputText, Modal, StatusAlert } from '@edx/paragon';
+import classNames from 'classnames';
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 
 import messages from './displayMessages';
-import './EditImageModal.scss';
+import styles from './EditImageModal.scss';
 import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
 import rewriteStaticLinks from '../../utils/rewriteStaticLinks';
 
 const LOADING_SPINNER_DELAY = 1000; // in milliseconds
 
+const imageDescriptionHelpLink = 'http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/accessibility/best_practices_course_content_dev.html#use-best-practices-for-describing-images';
```

Comment:
Ah, okay. That sounds good. The one thing I'm not sure of is that we're not using Redux for this component, which is how we access the studioContext elsewhere. The accessibility page doesn't subscribe to any Redux store either, and it gets necessary URLs via the accessibilityIndex.jsx file, which passes in the URL(s) as a [prop(s)](https://github.com/edx/studio-frontend/blob/master/src/accessibilityIndex.jsx#L36). Does that seem like an analogous situation to this one? Or should I figure out a way to get this documentation URL into this component through Redux?

Reply:
Hmm, I think the studio context is a global at this point, so that might be a way to access it--don't love that solution. Is the edit modal instantiated in Studio's mako template? Perhaps the prop pattern could be used here and Studio would be updated to pass along the prop. Does that work?